src/rgw/rgw_admin.cc

```diff
@@ -6807,7 +6807,7 @@ int main(int argc, const char **argv)
       result.reserve(NUM_ENTRIES);
 
       int r = store->getRados()->cls_bucket_list_ordered(
-	bucket_info, RGW_NO_SHARD,
+	bucket_info, RGW_NO_SHARD, RGW_DEFAULT_GENERATION,
```

Comment:
this is a command to list the current bucket index, so it should pass in `bucket_info.layout.current_index`